test/functional/wallet_assumeutxo.py

```diff
@@ -172,6 +182,13 @@ def run_test(self):
 
         self.log.info("Ensuring wallet can be restored from a backup that was created before the snapshot height")
         n1.restorewallet("w2", "backup_w2.dat")
+        # Check balance of w2 wallet
+        assert_equal(n1.getbalance(), 340)
+
+        # Check balance of w wallet after node is synced
+        w = n1.get_wallet_rpc("w")
+        n1.loadwallet("w")
+        assert_equal(w.getbalance(), 34)
```

Comment:
nit:  `w` would throw an error if used prior to loading the wallet. Should first load the wallet and secondly call `get_wallet_rpc`.

Reply:
done